### PR TITLE
fix actual eslint error #92

### PR DESCRIPTION
**Pull request by mdrianislam0or1**
### Description
noticed that when I run npm run eslint always says "0 Error, 0 Warnings" until when exists errors and warnings to fix.

Please include a summary of the change.

when I change the rules in the eslint.js file and run the command again. then I can see some change. if I change some code it gives the error and also identifies the error files.

fixes # (issue)

I put some screen shorts

#### Screenshots

1. After changing eslintrc.js

when i run npm run eslint
es1

2. create an error and save the file. run the command again.

es2

es3

If applicable, add screenshots about your changes

#### Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)

### How to test it

change the branch and go to the fix/#92 branch. run the command npm run eslint and do some mistakes and run the command again view the changes.

### Checklist:

- [x] Doesn't break the current code.

### CODE REVIEW

File: lefthook.yml [hunk 1]
```diff
@@ -1,50 +1,34 @@
...
+#     govet:
+#       tags: backend style
+#       files: git ls-files -m
+#       glob: "*.go"
+#       run: go vet {files}
+#   scripts:
+#     "hello.js":
+#       runner: node
+#     "any.go":
+#       runner: go run
```
Comment (heliomar-pena) [inline lefthook.yml]: Please uncomment lefthook file

File: package-lock.json [hunk 2]
```diff
@@ -18,6 +18,7 @@
         "graphql-tag": "~2.12.6",
         "react": "~17.0.2",
         "react-dom": "~17.0.2",
```
Comment (heliomar-pena) [inline package-lock.json]: Please remove package lock json changes from pull request

**Review by heliomar-pena — APPROVED** [review]
Good job! Please check comments about fix lefthook and remove package lock json, u can ask to me about it if you need

**Review by e-azocar — APPROVED** [review]
Good job, but, is necessary comment the file `lefthook.yml`?

File: lefthook.yml (hunk 1 above)
Comment (mdrianislam0or1) [reply lefthook.yml]: Done

File: lefthook.yml [hunk 3]
```diff
@@ -1,50 +1,34 @@
...
-      run: npm run stylelint:ci {staged_files}
-    spelling:
-      glob: '*.{js,jsx,md}'
-      run: npm run spelling:ci {staged_files}
-    markdown-link-check:
-      glob: '*.md'
-      run: npx markdown-link-check {staged_files}
+EXAMPLE USAGE
+Refer for explanation to following link:
+https://github.com/evilmartians/lefthook/blob/master/docs/full_guide.md
```
Comment (heliomar-pena) [inline lefthook.yml]: Is this configuration giving you an error?

File: package-lock.json (hunk 2 above)
Comment (mdrianislam0or1) [reply package-lock.json]: @Katsu08 Do you mean remove those

 "graphql-tag": "~2.12.6",
 "react": "~17.0.2",
"react-dom": "~17.0.2",

 three packages?

File: lefthook.yml (hunk 3 above)
Comment (mdrianislam0or1) [reply lefthook.yml]: No

Comment (mdrianislam0or1) [reply lefthook.yml]: i just uncomment lefthook file and push the code again.

Comment (heliomar-pena) [reply lefthook.yml]: Some things were deleted by mistake, you can copy the original content from dev branch and paste here and do the push again:

https://github.com/Azordev/maps-project/blob/dev/lefthook.yml

File: package-lock.json (hunk 2 above)
Comment (heliomar-pena) [reply package-lock.json]: I meant that package lock json was changed in this pull request, but that file doesn't should change hehe. You can copy the original content from dev branch and paste in your branch and push again:

https://github.com/Azordev/maps-project/blob/dev/package-lock.json

File: lefthook.yml (hunk 3 above)
Comment (mdrianislam0or1) [reply lefthook.yml]: Ok

**Review by heliomar-pena — APPROVED** [review]
Good work man!

**Pull request merged**